oaknational/Oak-Web-Application
# chore: update downloads to render icon LESQ-240

## Description

- Renders download Icon for supplementary resources

Downloads currently don't work as waiting for https://www.notion.so/oaknationalacademy/Connect-OWA-to-2023-content-downloads-server-13e214429a24440fa2dea124dacb5cee 

## Issue(s)

Fixes #LESQ-240

## How to test

1. Go to 
https://deploy-preview-1858--oak-web-application.netlify.thenational.academy/beta/teachers-2023/programmes/geography-secondary-ks4/units/test-unit/lessons/lesson-1-in-test-unit/downloads

3. Click on _______
4. You should see _______

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:

<img width="1264" alt="Screenshot 2023-09-01 at 16 06 24" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/ff96a3ae-72c5-4c4a-a5b1-de0bdc5d1c3a">


## Checklist
